src/components/EarnDashboard/components/StakingPositions/LpPositions.tsx

```diff
@@ -66,6 +70,15 @@ export const LpPositions: React.FC<ProviderPositionProps> = ({ ids, assetId }) =
         return
       }
 
+      mixpanel?.track(MixPanelEvents.ClickOpportunity, {
+        provider,
+        type,
+        assets: opportunity.original.underlyingAssetIds.map(assetId =>
+          GetCompositeAssetSymbol(assetId),
+        ),
```

Comment:
If you feel like using the shorthand
```suggestion
        assets: opportunity.original.underlyingAssetIds.map(GetCompositeAssetSymbolassetId)
```